Energy_Preserving.jl

```diff
@@ -0,0 +1,435 @@
+#This code is based on the Theorem 2 of the paper "Energy-Preserving Integrators and the Structure of B-series" (link: https://link.springer.com/article/10.1007/s10208-010-9073-1).
```

Comment:
Is this file still needed?

Reply:
No, its exactly the same as the one located in src. I will delete it